config.toml.in

```diff
@@ -9,3 +9,38 @@
 type = "file"
   [storage.settings]
   path = "$HOME/.pick/pick.safe"
+
+# Danger Area!
+# Only adjust the settings below this line if you know what you are doing.
+# If you change something and we update our recommended settings, your safe will
+# _not_ automatically upgrade to our new recommendations.
+# The encryption section controls the configuration of the safe encryption.
+#[encryption]
+# type let's you specify the encryption method to use.
+# Currently supported values: "aes_openpgp", "aes_gcm"
+#type = "aes_openpgp"
+#type = "aes_gcm"
+  # Settings for the "aes_openpgp" type
+  #[encryption.settings.aes_openpgp]
```

Comment:
I think since we're now using types instead of a map, we can drop the `Settings` field on `crypto.Config` and place `aes_openpgp`, `aes_gcm`, etc directly on `crypto.Config`.